### PR TITLE
Additional responses to reviewer comments (WIP)

### PR DESCRIPTION
**Pull request by rando2**
### Description of the proposed additions or changes

Second to last PR! Here is everything I could address without major reorganization of the text. The last one (which I'm hoping to knock out by Thursday) will essentially be a read through to address all of the remaining critiques. 

I wanted to request a review on these separately while it's still possible to see what I changed, since once I start reorganizing stuff, it will probably get a lot harder to review!

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

**Comment by agitter** [comment]
@rando2 I haven't been following all the pull requests and updates related to these revisions. Thanks a lot for working on them. Can you please let me know when you're getting close to resubmitting so I can review the changes?

**Comment by rando2** [comment]
@agitter Absolutely, I'll send an email with a diff before the winter holidays begin. We are supposed to resubmit by Jan 11, so I'm trying to get it out to authors as quickly as I can. I'm out of the office visiting family until Wednesday but will be back with lots of PRs on Thursday!

File: content/22.vaccines-trad.md [hunk 1]
```diff
@@ -458,6 +479,34 @@ In general, deciding on the prioritization and allocation of the COVID-19 vaccin
 Various frameworks, models, and methods have been proposed to tackle these issues with many countries, regions, or U.S. states devising their own distribution and administration plans [@doi:10.1126/science.abe6959; @doi:10.1038/s41598-020-78447-3; @doi:10.1016/j.vaccine.2020.12.059; @doi:10.1136/medethics-2020-107036; @doi:10.1101/2021.01.12.21249694].
 The majority of the distribution plans prioritized offering vaccines to key workers such as health care workers and those who are clinically vulnerable, such as the elderly, the immunocompromised, and individuals with comorbidities, before targeting the rest of the population, who are less likely to experience severe outcomes from COVID-19 [@url:https://ourworldindata.org/covid-vaccination-policy].
 The availability of vaccines developed in a variety of countries using a variety of platforms is likely to work in favor of worldwide vaccine access.
+The initiative by Texas Children’s Hospital and Baylor College of Medicine to develop Corbevax, a patent-free COVID-19 protein subunit vaccine, is an important step towards vaccine equity because the manufacturing specifications can be shared globally.
```
Comment (rando2) [inline content/22.vaccines-trad.md:482]: I'm not sure why your vaccine hesitancy text is showing up green, as that PR merged today! I don't think I changed it at all.

Comment (RLordan) [reply content/22.vaccines-trad.md:482]: Ummm I don't think I did anything but who knows...

**Review by RLordan — APPROVED** [review]
All good from my view @rando2

File: content/22.vaccines-trad.md (hunk 1 above)
Comment (rando2) [reply content/22.vaccines-trad.md:482]: Oh 100% not your fault @RLordan! When I work locally on my computer in git and then sync with GitHub, sometimes it seems like it takes a minute for it to catch up when the branch updates? But I don't know why that would happen.

**Pull request merged**